models/yang/sonic/sonic-bgp-common.yang

```diff
@@ -0,0 +1,346 @@
+module sonic-bgp-common {
+  namespace "http://github.com/Azure/sonic-bgp-common";
+  prefix sbc;
+
+  import ietf-inet-types {
+    prefix inet;
+  }
+
+  organization
+    "SONiC";
+
+  contact
+    "SONiC";
+
+  description
+    "SONIC BGP common YANG attributes for Neighbors and Peer group";
+
+  revision 2019-11-04 {
+    description
+      "Initial revision.";
+  }
+
+  typedef bgp_peer_type {
+    type enumeration {
+      enum INTERNAL {
+        value 0;
```

Comment:
Do we want to get rid of the values for INTERNAL & EXTERNAL and just use the strings?